client/assets/images/check_list.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please optimize sgv image using this service: https://jakearchibald.github.io/svgomg/